### ISSUE TITLE
Add Vagrant and/or Dockerfile

### ISSUE THREAD

**Issue by steffenmllr** [jmapio/jmap-perl#2]
More a feature request than a bug: It would be great if you could add a Vagrant file // Dockerfile to get the proxy up and running locally if people - like me - just want to test it out.

**Comment by brong** [comment]
Yeah, I would like to do that. I want to port from sqlite to something
more robust first, is really not handling even modest load very well.

On Fri, Aug 21, 2015, at 19:21, Steffen Müller wrote:

> More a feature request than a bug: It would be great if you could add
> a Vagrant file // Dockerfile to get the proxy up and running locally
> if people - like me - just want to test it out.
> 
> — Reply to this email directly or view it on GitHub[1].

## 

  Bron Gondwana
  brong@fastmail.fm

Links:
1. https://github.com/jmapio/jmap-perl/issues/2

**Comment by robn** [comment]
This is on me. I already have half a Dockerfile. No ETA yet though.

**Comment by steffenmllr** [comment]
Something other than sqlite would be great - how does jmap-perl compare to implementations like https://github.com/nylas/sync-engine ? So far those are the only two projects that provide a sane rest api for a email account when dealing with legacy mail system.

**Comment by thomas-hilaire** [comment]
Hello,

This is a late reply but it can help people reading this issue, we provide a jmap-perl docker image at https://hub.docker.com/r/rpignolet/jmap-perl/

Try it!

**Comment by thomascube** [comment]
The docker images published by rpignolet no longer worked for me.
Added Dockerfile in #53 which builds with the latest version from this repo.
